customer-portal-app/src/container/about-us/index.js

```diff
@@ -1,64 +1,133 @@
-                                    <strong>IBAN t.n.v.</strong>
-                                </td>
-                                <td>Energie Dongen</td>
-                            </tr>
-                            <tr>
-                                <td>
-                                    <strong>BTW nummer</strong>
-                                </td>
-                                <td>987987.097809.0890</td>
-                            </tr>
-                        </tbody>
-                    </Table>
-                </Col>
-            </Row>
+            {state.isLoading ? (
+                <LoadingView />
+            ) : (
+                <Row>
+                    {state.result.map(item => (
+                        <Col xs={12} lg={6} className={'mb-3'}>
+                            <Card>
+                                <Card.Header className={'card-header_title'}>Informatie over {item.name}</Card.Header>
+                                <Card.Body>
+                                    <Table responsive>
+                                        <tbody>
+                                            <tr>
+                                                <td>
+                                                    <strong>Naam</strong>
+                                                </td>
+                                                <td>{item.name}</td>
+                                            </tr>
+                                            <tr>
+                                                <td>
+                                                    <strong>Adres</strong>
+                                                </td>
+                                                <td>{item.address}</td>
+                                            </tr>
+                                            <tr>
+                                                <td>
+                                                    <strong>Postcode / Plaats</strong>
+                                                </td>
+                                                <td>
+                                                    {item.postalCode} {item.city}
+                                                </td>
+                                            </tr>
+                                            <tr>
+                                                <td>
+                                                    <strong>Kvk</strong>
+                                                </td>
+                                                <td>{item.kvkNumber}</td>
+                                            </tr>
+                                            <tr>
+                                                <td>
+                                                    <strong>Website</strong>
+                                                </td>
+                                                <td>http://energiedongen.nl</td>
```

Comment:
This hard value of website URL must be changed to {item.website}. But that value is now missing in API for related-administration. Field administration website must be added to that API also.